### ISSUE TITLE
Improve link styles (a11y)

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-minimalist#19]
Underlines (or some other type of non-color dependent distinguishing style) should be added for inline links.

Link to body text contrast should be at least 3:1, it's currently 2:1, also problematic on hover.
